### ISSUE TITLE
Recipe button freezing in one choice after one craft

### ISSUE THREAD

**Issue by Toma400** [TheIllusiveC4/Polymorph#152]
#### Versions (Be specific, do not write "latest"):
* Polymorph: 0.38
* Forge/Fabric: Forge 1.16.5
#### Steps to Reproduce:
1. Add mods that conflict with each other quite a lot (BetterNether & Infernal Expansion is common for me)
2. Try making one of their conflicting recipes (for example: BN conflicts with vanilla when making polished basalt)
3. After successful crafting, try making another crafting attempt without exiting the GUI
4. GUI won't show up button for output switching, if you needed to switch it earlier for exact recipe (I didn't test that for different recipes yet)

Crash Log: Please use a paste site like [hastebin](https://hastebin.com/).

**Comment by TheIllusiveC4** [comment]
I cannot reproduce your issue. Here are my steps:

1. I installed BetterNether Reforged alongside Polymorph.
2. I open the vanilla Crafting Table and place 4 vanilla Basalt blocks as needed for the vanilla Polished Basalt recipe.
3. Polymorph identifies a conflict and shows the vanilla Polished Basalt and BN's Basalt Stalactite as possible outputs.
4. The Basalt Stalactite was the default, so I changed it to the Polished Basalt and crafted it.
5. Without leaving the GUI, I place another 4 vanilla Basalt blocks into the Crafting Table and the button still appears. I am still able to craft another set of Polished Basalt.
6. I am also able to craft any other recipe with conflicts without leaving the GUI.

Are you able to reproduce this issue with _only_ BetterNether and Polymorph installed? If not, this is likely a mod conflict and you'll need to find the mod that is causing the issue before I can investigate further.
